### PR TITLE
Added autofocus in a number of forms

### PR DESCRIPTION
**Pull request by Astradeni**
Added autofocus in a number of forms, plus correction of a few spelling mistakes

### CODE REVIEW

File: app/views/admin/campaigns/_form.html.haml [hunk 1]
```diff
@@ -11,7 +11,7 @@
   .col-md-8
     = semantic_form_for(@campaign, url: (@campaign.new_record? ? admin_conference_campaigns_path : admin_conference_campaign_path(@conference.short_title, @campaign))) do |f|
       = f.inputs do
-        = f.input :name
+        = f.input :name, input_html: { autofocus: true }
```
Comment (Ana06) [inline app/views/admin/campaigns/_form.html.haml]: Is the `input_html` needed?  I mean, this should work as well:

``` Ruby
= f.input :name, autofocus: true
```

Doesn't it? 🤔

Comment (differentreality) [reply app/views/admin/campaigns/_form.html.haml]: It does not.

File: app/views/admin/commercials/index.html.haml [hunk 2]
```diff
@@ -14,7 +14,7 @@
   .row
     .col-md-6
       = semantic_form_for(@commercial, url: admin_conference_commercials_path(conference_id: @conference.short_title)) do |f|
-        = f.input :url, label: 'URL', as: :string, input_html: { required: 'required' },
+        = f.input :url, label: 'URL', as: :string, input_html: { required: 'required' }, input_html: { autofocus: true },
```
Comment (bear454) [inline app/views/admin/commercials/index.html.haml]: Please fold this into the existing hash. For example:

```
..., input_html: { required: 'required', autofocus: true }, ...
```

**Comment by Astradeni** [comment]
Done... Thanks for the suggestion!

screenshot_20180104_011304

**Comment by differentreality** [comment]
Thank you @Astradeni :+1:

**Pull request merged**